src/Polly.Core/Telemetry/ResilienceStrategyTelemetry.cs

```diff
@@ -21,6 +21,22 @@ internal ResilienceStrategyTelemetry(ResilienceTelemetrySource source, Telemetry
 
     internal ResilienceTelemetrySource TelemetrySource { get; }
 
+ #pragma warning disable S4049
```

Comment:
Rather than suppress, I suggest words such as `As`, `To`, `Create` or `Convert` instead of `Get`.